README.md

```diff
@@ -6,13 +6,23 @@ SwiftFrame is a command line application for composing framed screenshots. Here'
 
 ## Usage
 
-For the complete set of options please see
-
-```
-swiftframe -h
-```
-
-A working example is set up inside the `Example` folder.
+Configuration is mostly possible through command-line arguments (see `swiftframe -h` for further information). However, in most cases it's more convenient to provide a configuration file. The latter is a plain JSON file supporting the following options:
+
+- `background` – How to fill the canvas background. Colors are expected to specified in hex form (CSS-shorthand notation supported). Possible settings:
+  - Solid color in CSS notation, e.g. `"#FFFFFF"`
+  - Linear gradient in CSS notation, e.g. `"linear-gradient(to right bottom, #000, #FFF)"`
+- `titles` – Object detailing the rendering of titles. Child options:
+  - `color` – Title color in CSS notation, e.g. `"#000"`
+  - `font` – (Relative or absolute) path to a the font file
+  - `padding` – Padding in pixels used to surround the title. Components separated by space in the order `"TOP LEFT BOTTOM RIGHT"`, e.g. `"5 10 5 10"`.
```

Comment:
Yeah, I know. :D

I guess in most cases left and right are gonna have the same value though so the order wouldn't matter then.